radix_sort.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import unicode_literals
+from math import floor, log10
+from time import time
+
+
+def radix_sort(initial_list):
+    order_mag = _order_mag(initial_list)
+    return_list = initial_list[:]
+    for i in range(order_mag+1):
+        buckets = [[] for x in range(10)]
+        for number in return_list:
+            buckets[(number // (10**i)) % 10].append(number)
```

Comment:
Here you are raising 10 to the `i`th power once for every single number in the incoming list, for every single value of `i`.  Do you need to do this operation that many times?  In this line of code, the only value that changes from time to time is `number` (within a single value of `i`).  Find efficiencies where you can.